dev-db/mysql++/mysql++-3.3.0.ebuild

```diff
@@ -37,24 +40,25 @@ src_prepare() {
 	# Current MySQL libraries are always with threads and slowly being removed
 	sed -i -e "s/mysqlclient_r/mysqlclient/" "${S}/configure" || die
 	rm "${S}/doc/"README-*-RPM.txt || die
+	multilib_copy_sources
 }
 
-src_configure() {
+multilib_src_configure() {
 	local myconf=(
 		--enable-thread-check
 		--with-mysql="${EPREFIX}/usr"
 		--with-mysql-lib="${EPREFIX}$(mysql_config --variable=pkglibdir)"
 		--with-mysql-include="${EPREFIX}$(mysql_config --variable=pkgincludedir)"
 	)
-	econf "${myconf[@]}"
+	ECONF_SOURCE="${S}" econf "${myconf[@]}"
```

Comment:
You shouldn't need ECONF_SOURCE if you're doing multilib_copy_sources. It is better if it works without multilib_copy_sources.

Reply:
This multilib package is needed by a 32 bit Custom application.
Tested with multilib_copy_sources only, but unfortunately does not build (make needs configure.ac)
Removed unneeded ECONF_SOURCE